ckanext/dia_theme/less/dia_new.less

```diff
@@ -378,6 +442,12 @@ div {
   }
 }
 
+@media (max-width: 400px) {
```

Comment:
Are there other existing media queries at 400px? I'm a bit wary of adding more variety to these, it seems like there are a lot already.

I'd have thought we really only need 2 levels, mobile (768?) and tablet (980?). What levels do ckan core use I wonder? They're probably the ones we should use too.

Reply:
Totally understand where your coming from. Looks of the layout transitions have been complicated by the many different sizes that exist in default CKAN. That being said I specifically used 400 for the transition because there is plenty of room for the "Order By" label before this. The lowest CKAN media is 768px. Seams premature to be removing the label at this point.

Se below how it looks if we change this rule to 768px. Happy to update if you still think its the best approach. 
![Screen Shot 2021-09-02 at 09 29 25](https://user-images.githubusercontent.com/87961621/131747882-18f336cb-2152-457f-a16a-740a684c0be2.png)